api/HTMLInputElement.json

```diff
@@ -1224,13 +1321,11 @@
             "firefox": {
               "version_added": "16",
               "notes": "Experimental, and without specific UI. There are still differences with the latest spec: <a href='https://bugzil.la/835773'>bug 835773</a>.",
-              "flags": [
```

Comment:
I think your editor might've autoformatted this, which is contrary to our linter. This change can be reverted

Reply:
I didn't pay attention, it was corrected.